lms/templates/instructor/instructor_dashboard_2/cohort-discussions.underscore

```diff
@@ -0,0 +1,49 @@
+<div class="cohort-discussions-nav">
+    <form action="" method="post" id="cohort-coursewide-discussions-form" class="cohort-coursewide-discussions-form">
+        <h3 class="subsection-title"><%- gettext('Designate discussion topics as divided by cohort') %></h3>
```

Comment:
@catong I think that's just an implemented error in the wireframes. @muzaffaryousaf, please change the link text and the heading text once the link is clicked to this text: "Specify whether discussion topics are divided by cohort".